accepted/module-system.md

```diff
@@ -1355,6 +1355,13 @@ type `type`, and an [import context](#import-context) `import`:
 * Otherwise, if `member-uses` contains a single module, return the member of
   type `type` named `name` in that module.
 
+* Otherwise, if the implementation defines a global member `member` of type
+  `type` named `name`, return that member.
+
+  > This includes the global functions and mixins defined as part of the Sass
+  > spec, and may also include other members defined through the
```

Comment:
Are the names of global functions and mixins defined in the Sass spec protected in another part of the spec? Can modules override those methods if they want since these are returned last?

Reply:
No, they aren't protected. It's totally valid to define a member with the same name as a global member, and if you do it overrides the global member definition.